src/main/java/com/esri/samples/scene/animate_3d_graphic/Animate3dGraphicController.java

```diff
@@ -115,42 +105,45 @@ public void initialize() {
       GraphicsOverlay mapOverlay = new GraphicsOverlay();
       mapView.getGraphicsOverlays().add(mapOverlay);
 
-      // create renderer to handle updating plane heading using the graphics card
+      // create renderer to rotate the plane graphic in the mini map
       SimpleRenderer renderer2D = new SimpleRenderer();
+      SimpleMarkerSymbol plane2DSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, 0xFF0000FF, 10);
+      renderer2D.setSymbol(plane2DSymbol);
       renderer2D.setRotationExpression("[ANGLE]");
       mapOverlay.setRenderer(renderer2D);
 
-      // set up route graphic
+      // create a placeholder graphic for showing the mission route in mini map
       SimpleLineSymbol routeSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 2);
       routeGraphic = new Graphic();
       routeGraphic.setSymbol(routeSymbol);
       mapOverlay.getGraphics().add(routeGraphic);
 
-      // create 2D and 3D plane graphics
-      plane2D = create2DPlane();
+      // create a graphic with a blue (0xFF0000FF) triangle symbol to represent the plane on the mini map
+      Map<String, Object> attributes = new HashMap<>();
+      attributes.put("ANGLE", 0f);
+      plane2D = new Graphic(new Point(0, 0, WGS84), attributes);
       mapOverlay.getGraphics().add(plane2D);
-      plane3D = create3DPlane();
+
+      // create a graphic with a ModelSceneSymbol of a plane to add to the scene
+      String modelURI = new File("./samples-data/bristol/Collada/Bristol.dae").getAbsolutePath();
+      ModelSceneSymbol plane3DSymbol = new ModelSceneSymbol(modelURI, 1.0);
+      plane3DSymbol.loadAsync();
+      plane3D =  new Graphic(new Point(0, 0, 0, WGS84), plane3DSymbol);
       sceneOverlay.getGraphics().add(plane3D);
 
+      // create an orbit camera controller to follow the plane
+      orbitCameraController = new OrbitGeoElementCameraController(plane3D, 20.0);
+      sceneView.setCameraController(orbitCameraController);
```

Comment:
Not sure if this is the best code, but the following line makes the horizon visible when the aircraft starts and I think its a better initial view point:

```
orbitCameraController.setCameraPitchOffset(75);
```